stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberFormatter.kt

```diff
@@ -495,3 +496,12 @@ internal sealed class PhoneNumberFormatter {
         )
     }
 }
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun convertPhoneNumberToE164(
+    nationalPhoneNumber: String,
+    countryCode: String,
+): String? {
+    val formatter = PhoneNumberFormatter.forCountry(countryCode)
+    return formatter.toE164Format(nationalPhoneNumber)
+}
```

Comment:
Made this one method accessible so that we don't have to restrict all the `PhoneNumberFormatter` nested classes.